config/helm/chart/default/templates/Common/webhook/deployment-webhook.yaml

```
@@ -56,15 +56,13 @@ spec:
           whenUnsatisfiable: ScheduleAnyway
           labelSelector:
             matchLabels:
-              app.kubernetes.io/component: webhook
-              app.kubernetes.io/name: dynatrace-operator
+              {{- include "dynatrace-operator.webhookLabels" . | nindent 14 }}
         - maxSkew: 1
           topologyKey: "kubernetes.io/hostname"
           whenUnsatisfiable: DoNotSchedule
           labelSelector:
             matchLabels:
-              app.kubernetes.io/component: webhook
-              app.kubernetes.io/name: dynatrace-operator
+              {{- include "dynatrace-operator.webhookLabels" . | nindent 14 }}
```

```suggestion
             {{- include "dynatrace-operator.webhookSelectorLabels" . | nindent 14 }}
```

so we don't have the version in it, maybe could cause problem during upgrades 
- the old version's pods will not match ==> topologySpreadConstraints has no affect (probably)